### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by practicalswift**
Fix typos.

### CODE REVIEW

**Comment by PlasmaPower** [comment]
The folders other than `rai` are libraries. I think it's better to leave those alone.

File: FindCryptoPP.cmake [hunk 1]
```diff
@@ -16,7 +16,7 @@
 #
 #   CRYPTOPP_LIBRARIES
 #     Points to the CryptoPP libraries that should be passed to
-#     target_link_libararies.
+#     target_link_libraries.
```
Comment (PlasmaPower) [inline FindCryptoPP.cmake]: This is another library.

**Comment by practicalswift** [comment]
@PlasmaPower Fixed. Please re-review :-)

**Comment by PlasmaPower** [comment]
LGTM though I don't have permission to merge. @androm3da ?

**Pull request merged**